### PR TITLE
Added debug input 

### LINKED ISSUE #3749: Dread: Add to faq about the item refresh combo
Regarding speedbooster, cloak, etc

### PR DESCRIPTION
**Pull request by Hugoshido**
into Dread-FAQ

Fixes #3749

### CODE REVIEW

**Review by henriquegemignani — CHANGES_REQUESTED** [review]
Use spaces, not tabs.

**Comment by Hugoshido** [comment]
> Use spaces, not tabs.
Changed.
Didn´t know this makes a difference.

File: randovania/games/dread/game_data.py [hunk 1]
```diff
@@ -88,6 +88,12 @@ def _generator() -> game.GameGenerator:
             "* Other mods likely work fine.\n\n"
             "When reporting issues, your first step is always to reproduce the issue without mods, "
             "**no matter how simple** the mod is."
+        ),
+		(
```
Comment (henriquegemignani) [inline randovania/games/dread/game_data.py:92]: you missed this one

**Review by henriquegemignani — CHANGES_REQUESTED** [review]

**Comment by henriquegemignani** [comment]
> Changed. Didn´t know this makes a difference.

Python is whitespace sensitive, so there can be weirdness with it.

But no matter the project, mixing tabs and spaces is a no-go.

**Comment by Hugoshido** [comment]
How do you see this in the code?

**Comment by henriquegemignani** [comment]
> How do you see this in the code?

Most IDEs offer some way of rendering whitespace somehow. Sometimes selecting it also works reasonably well.
You also just configure your IDE to output the correct indentation to the project and just don't worry about it.

When reviewing PRs in GitHub, a tab shows up as 8 spaces so it's easy to see. Selecting it also makes it obvious it's a tab instead of spaces.

**Review by henriquegemignani — APPROVED** [review]

**Pull request merged**